db.go

```diff
@@ -121,13 +121,13 @@ func getAll(teamName, imageName string) []scoreEntry {
 
 	if imageName != "" {
 		fmt.Println("image specificed, searching for all records ")
-		cursor, err = coll.Find(context.TODO(), bson.D{{"team.id", teamObj.Id}, {"image", getImage(imageName)}}, findOptions)
+		cursor, err = coll.Find(context.TODO(), bson.D{{"team.id", teamObj.ID}, {"image", getImage(imageName)}}, findOptions)
 		if err != nil {
 			panic(err)
 		}
 	} else {
-		fmt.Println("no imag, seaaaaarrchchhin", teamObj.Id, imageName)
-		cursor, err = coll.Find(context.TODO(), bson.D{{"team.id", teamObj.Id}}, findOptions)
+		fmt.Println("no imag, seaaaaarrchchhin", teamObj.ID, imageName)
```

Comment:
if ur gonna give some love sarpedon fix this plz

Reply:
done